enclave/core/sgx/cpuid.c

```diff
@@ -10,6 +10,23 @@
 #include "platform_t.h"
 
 static uint32_t _cpuid_table[OE_CPUID_LEAF_COUNT][OE_CPUID_REG_COUNT];
+static const uint32_t _cpuid_leaves[OE_CPUID_LEAF_COUNT] = {0,
```

Comment:
Since we are creating a mapping for supported leaves here, can this just be reduced to the set of emulated leaves only? It also simplifies the logic for `oe_is_emulated_cpuid_leaf` to have it simply iterate through the leaves in `_cpuid_leaves` instead of maintaining a separate bit of logic that can get out of sync.

Reply:
Thanks for the comments. Fixed in the latest commit.